kong/plugins/apifortress/base64.lua

```diff
@@ -0,0 +1,86 @@
+-- ©2016 API Fortress Inc.
+-- Base64-encoding
+-- Sourced from http://en.wikipedia.org/wiki/Base64
+require('math')
+
+local index_table = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/'
+
+
+function to_binary(integer)
```

Comment:
Lua uses the "global by default" paradigm, so all these functions end up as globals. Then again, please remove the entire file, and use https://github.com/openresty/lua-nginx-module#ngxencode_base64 instead